Fluid.Tests/ArrayFiltersTests.cs

```diff
@@ -371,6 +371,55 @@ public async Task Where()
             Assert.Single(result3.Enumerate(context));
         }
 
+        [Fact]
+        public async Task WhereWithTruthy()
+        {
+            var input = new ArrayValue(new[]
+            {
+                new ObjectValue(new { Title = "a", Pinned = true, Missing = 1 }),
+                new ObjectValue(new { Title = "b", Pinned = false }),
+                new ObjectValue(new { Title = "c", Pinned = true, Missing = 1 })
+            });
+
+            var options = new TemplateOptions();
+            var context = new TemplateContext(options);
+            options.MemberAccessStrategy.Register(new { Title = "a", Pinned = true }.GetType());
+            options.MemberAccessStrategy.Register(new { Title = "a", Pinned = true, Missing = 1 }.GetType());
+
+            // x | where: "Missing"
```

Comment:
What would be the behavior for `where: Pinned`, it's a boolean, I would expect 2. Can you add the test?

Reply:
It is 2 as you expect. The `Where` test (above `WhereWithUndefined`) already performs that check.